### PR TITLE
exit with staus code 1 if decaf fails

### PR DESCRIPTION
**Pull request by kyleholzinger**
It should probably not exit with status code 0

### CODE REVIEW

File: bin/decaf [hunk 1]
```diff
@@ -72,5 +72,6 @@ function runByFile(file, config) {
   } catch (e) {
     console.log(chalk.red('Skipping file "%s" because of error.'), file);
     console.log(chalk.red(e.stack));
+    process.exit(1)
```
Comment (bouk) [inline bin/decaf]: I think the point of this is that it keeps on processing other files even though this one failed. Maybe put `exit(1)` at the end if any file failed?

Comment (kyleholzinger) [inline bin/decaf]: Good idea! Changed

**Comment by kyleholzinger** [comment]
@bouk Any other comments on this? It'd be awesome to get this in, currently my team is blocked on this and would love to be able to use your npm package

**Comment by kyleholzinger** [comment]
bump @bouk

**Comment by kyleholzinger** [comment]
@juliankrispel Tagging you since you were the last person to commit

**Comment by kyleholzinger** [comment]
Anyone want to merge this in? Been sitting here for quite a while

**Comment by juliankrispel** [comment]
Hey sry will look at it in the morning - thx

Julian Krispel-Samsel
rainforestqa.com
goodafternoon.co

**Comment by juliankrispel** [comment]
Looked now and merged - thx a ton!

**Comment by kyleholzinger** [comment]
Thanks man! No worries

**Pull request merged**